sdk/src/wallet/operations/block.rs

```diff
@@ -14,23 +14,32 @@ where
 {
     pub(crate) async fn submit_basic_block(
         &self,
-        payload: Option<Payload>,
+        payload: impl Into<Option<Payload>> + Send,
         issuer_id: impl Into<Option<AccountId>> + Send,
+        allow_negative_bic: bool,
```

Comment:
Should this be optional with default false? For the bindings this would be nicer to use then